data/human/culture conversations.txt

```diff
@@ -615,3 +615,68 @@ mission "Sol-Bound Shuttle"
+					decline
+			`	Every step takes significantly more effort than you're used to, but you eventually reach the crowd. From the signs that have been put up on the way it looks like the 'Burthen Triathlon' is taking place today, and the spaceport has been designated the finishing point. A couple of especially sturdy-looking marquees have been set up; the smaller one appears to be the administration tent, currently only accessible to event staff, while the other one is far larger and is catering food and drinks for the crowd nearby.`
+			choice
+				`	(Walk over to the catering tent.)`
+				`	(Stay outside.)`
+					goto outside
+				`	(Return to your ship.)`
+					decline
+
+			`	Thankfully the walk to the catering tent is far shorter than the walk from your ship, but nevertheless you're exhausted by the time you drag your feet inside. A lot of effort has been put into this tent: the metal chairs are all sturdy to allow offworlders not used to the gravity to pull themselves up, but cushioned as to not make it uncomfortable to sit in; a small firepit has been built in the middle of the tent, along with a couple fire extinguishers placed nearby; and the whole place is brightly lit in contrast to Burthen's typically chilly weather. Quite a few people, both Burthensiders and offworlders, are mingling about here.`
+			`	There's a counter in the corner with two levels, apparently so that the Burthensider attending it can serve both offwolders and locals. Alongside the typical drinks you've seen in spaceport bars as well as some ones local to the southern Core, the counter also offers a local drink called the 'Antigrav' that claims to "ease visitors from Burthen's natural pull", and comes in a specially made glass, all for 30 credits.`
+			branch "no credits"
+				credits < 30
+			choice
+				`	(Order a drink from the counter.)`
+				`	(Order the 'Antigrav' from the counter.)`
+					goto antigrav
+				`	(Don't order anything and leave the tent.)`
+					goto "leave tent"
+
+			action
+				payment -2
+			`	You hand over a couple credits for the drink and leave the tent, sipping with some difficulty as you go; the Burthensider at the counter advises you not to drink too much of it, and not to drop the cup "unless you want a munted foot for a few weeks."`
+				goto "leave tent"
+
+			label antigrav
+			action
+				payment -30
+				outfit "Antigrav Glass"
+			`	It turns out the glass the 'Antigrav' comes with has a miniature repulsor installed so that you don't need to hold it while drinking it; the 'Antigrav' itself is a sort of highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in 'Westland', the planetside country the spaceport is located in. The Burthensider at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`
```

Comment:
```suggestion
			`	It turns out the glass the Antigrav comes with has a miniature repulsor installed so that you don't need to hold it while drinking it. The drink itself is a sort of highly carbonated soda that tastes like something in between lingonberry and raspberry, made from the native berries found in Westland, the planetside district the spaceport is located in. The Burthensider at the counter advises you not to drink too much of it at once and to not turn it off while it's levitating, "lest it falls on your foot and we have to treat your bone fractures."`
```

Just because you _can_ technically write out a full paragraph in one sentence using semicolons doesn't mean it's a good idea. The word "country" implies either autonomy or rurality, at least to me, neither of which seem appropriate in this context.

Is there a specific type of "native berry" grown in Westland?

Reply:
1. Err, I did write this one funny. It was almost a stream of consciousness thing, too bad I have to follow the rules here. "Planetary district" is probably the right word to use as well, I forgot about that one.

2. Yes.